app/helpers/brexit_checker_helper.rb

```diff
@@ -12,7 +12,13 @@ def criteria_aside_label(heading = nil)
   end
 
   def filter_items(items, criteria_keys)
-    items.select { |i| i.show?(criteria_keys) }
+    filtered = items.select { |i| i.show?(criteria_keys) }
+    sorted_items(filtered)
```

Comment:
Minor: we don't really need this method. It's a single-use one-liner that contains a clear call to `sort_by`.